presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -441,14 +442,9 @@ public static void main(String[] args)
         Logging.initialize();
 
         Optional<Path> dataDirectory = Optional.empty();
-        if (args.length > 0) {
-            if (args.length != 1) {
-                log.error("usage: HiveQueryRunner [dataDirectory]\n");
-                log.error("       [dataDirectory] is a local directory under which you want the hive_data directory to be created.]\n");
-                System.exit(1);
-            }
-
-            File dataDirectoryFile = new File(args[0]);
+        Optional<String> suppliedDataDirectoryPath = getProperty("DATA_DIR");
+        if (suppliedDataDirectoryPath.isPresent()) {
+            File dataDirectoryFile = new File(suppliedDataDirectoryPath.get());
             if (dataDirectoryFile.exists()) {
```

Comment:
Extract a common method to validate the data directory.

Reply:
Done.